test_sorting_insertionsort.py

```diff
@@ -15,3 +15,44 @@ def test_randoms():
 def test_string():
     with pytest.raises(TypeError):
         insertion_sort('cat')
+
+
+def test_empty():
+    x = []
+    y = x
+    insertion_sort(x)
+    assert x == y
+
+
+def test_reversed():
+    x = []
+    for i in range(1, 10):
+        x.append(i)
+    y = []
+    for i in reversed(x):
+        y.append(i)
+    insertion_sort(y)
+    z = y
+    assert z == x
+
+
+def test_ordered():
+    x = []
+    for i in range(1, 10):
+        x.append(i)
+    insertion_sort(x)
+    z = x
+    assert z == x
+
+
+def test_various_length():
+    x = [1, 200, 600, 1234567, 99999, 7]
+    insertion_sort(x)
+    y = x
+    expected = [1, 7, 200, 600, 99999, 1234567]
+    assert expected == y
```

Comment:
Other than the error check in your first test, this is the first test that is actually doing anything.